deploy/terraform/run/sap_system/variables_global.tf

```
@@ -15,6 +15,10 @@ variable "infrastructure" {
 
 variable "jumpboxes" {
   description = "Details of the jumpboxes"
+  default     = {
+    windows = {},
+    linux = {}
```

My bad, those two should be list:
```suggestion
    windows = [],
    linux = []
```